src/MOF/read.jl

```diff
@@ -7,13 +7,14 @@ function MOI.read_from_file(model::Model, io::IO)
         validate(io)
     end
     object = JSON.parse(io; dicttype=Object)
-    if object["version"] > VERSION
+    file_version = _parse_mof_version(object["version"])
+    if file_version > VERSION
         error("Sorry, the file $(filename) can't be read because this library" *
-              " supports version $(VERSION) of MathOptFormat, but the file " *
-              "you are trying to read is version $(object["version"]).")
+              " supports v$(VERSION) of MathOptFormat, but the file you are " *
```

Comment:
Minor thing: spacing not consistent within the quotes (line 12: no space at the end of the line; line 13: space).

Reply:
I don't have a consistent rule for this. Something to think about.